sphericart-torch/setup.py

```diff
@@ -8,7 +8,17 @@
 
 
 ROOT = os.path.realpath(os.path.dirname(__file__))
-SPHERICART_ARCH_NATIVE = os.environ.get("SPHERICART_ARCH_NATIVE", "ON")
+
+
+def get_arch_native():
+    # may not want to build processor-specific instructions
+    SPHERICART_ARCH_NATIVE = os.environ.get("SPHERICART_ARCH_NATIVE", "ON")
+    if SPHERICART_ARCH_NATIVE == "ON" and (
+        "bdist_wheel" in sys.argv or "wheel" in sys.argv
+    ):
+        return "OFF"
```

Comment:
this looks like the wrong logic: (a) we should detect the `build_ext` step instead of the wheel one; and (b) this means we will never use `SPHERICART_ARCH_NATIVE`, even when building locally. Instead I would try to detect CIBUILDWHEELS (or however this is called), i.e. detect the environment in which we are building wheels for large scale distribution

Reply:
OK well since it's not clear to me yet how we're going to be building wheels for distribution just yet (which will be a separate PR), do you think it's reasonable to instead set:

`SPHERICART_ARCH_NATIVE = os.environ.get("SPHERICART_ARCH_NATIVE", "ON")`

to OFF by default, or would this be too annoying to deal with documentation wise?

Reply:
not sure how I missed this, sorry. Yes `SPHERICART_ARCH_NATIVE` in the environment would be fine!

Reply:
I've now set SPHERICART_ARCH_NATIVE to OFF by default in all setup.py files.